src/GuidesIndex.php

```diff
@@ -25,6 +25,24 @@ private static function getSummaries(
     return require(BuildPaths::GUIDES_SUMMARY);
   }
 
+  public static function search(string $term, SearchResultSet &$results): void {
```

Comment:
No references in hack.

Alternatively, return a vector of results, or if you have different tyeps of results, return your own searchresultset, and do something like:

$combined_results->addResults(Foo::search($term));